packages/fastify-graphiql/Readme.md

```diff
@@ -13,7 +13,10 @@ import fastifyGraphiql from 'fastify-graphiql';
 // Setting up webserver
 const app = Fastify();
 
-app.register(fastifyGraphiql);
+app.register(fastifyGraphiql, {
+   url: '/graphiql',
```

Comment:
not /api/ in front?

Reply:
It could of course be `/api/` in front, but your commenting on an example in the Readme-file. It's hardly relevant here.